src/Umbraco.Web.Common/Extensions/UmbracoCoreServiceCollectionExtensions.cs

```diff
@@ -156,6 +158,23 @@ public static IServiceCollection AddUmbracoCore(this IServiceCollection services
             return services;
         }
 
+        public static IServiceCollection AddUmbracoRequestLocalization(this IServiceCollection services)
+        {
+            services.Configure<RequestLocalizationOptions>(options =>
```

Comment:
So that we don't conflict with global aspnet things i think we need to sub-class this so that it's BackOfficeRequestLocalizationOptions just like we do for all other services. This will also mean looking at the RequestLocalizationMiddleware  since we may need to sub-class that and have our own as well. We can't just globally set these options since this affects every request that goes to a site, we need to only apply these types of things to our own requests and not affect global settings. Just like everything else the aspnetcore team hasn't made that perfectly flexible wo we'll need to work around it. 

I'm happy to look into this tonight?

Reply:
You are right - Forgot about that.. You are very welcome to make the changes 💪